app/templates/directives/iuclid-form/iuclid-form-contents.html

```diff
@@ -0,0 +1,16 @@
+<div ng-repeat="content in contents" ng-switch="content.type">
+    <iuclid-block ng-switch-when="block" content="content"></iuclid-block>
+    <iuclid-text ng-switch-when="text" content="content"></iuclid-text>
+    <!--
+        I think for all our form attributes we should just pass it the whole content object
+        Check my <iuclid-block> and <iuclid-text> implementations for how to pass the whole object
+     -->
+    <iuclid-pick-list ng-switch-when="picklist" code="{{ content.phrasegroup }}" content="content"></iuclid-pick-list>
+    <!--
+        The template for adding new form types is like this:
+        <iuclid-$type ng-switch-when="$type" content="content" ></iculid-$type>
+    -->
```

Comment:
Like I commented here, this is where you can add new iuclid form attributes as you create them.
For instance if you add a `numeric` you could add it here like this:
```
<iuclid-numeric ng-switch-when="numeric" content="content"></iuclid-numeric>
```